src/party.cpp

```diff
@@ -419,7 +421,7 @@ void Party::shareExperience(uint64_t experience, Creature* source/* = nullptr*/)
 	leader->onGainSharedExperience(shareExperience, source);
 }
 
-bool Party::canUseSharedExperience(const Player* player) const
+bool Party::canUseSharedExperience(const Player* player)
```

Comment:
But we would not know which ``if`` condition was evaluated to ``false`` in ``Party::canUseSharedExperience``, it could be either because of [this](https://github.com/raymondtfr/forgottenserver/blob/bd6c492d60f00bbb258042f5a6ffd0e3dc03c9d7/src/party.cpp#L426), or [this](https://github.com/raymondtfr/forgottenserver/blob/bd6c492d60f00bbb258042f5a6ffd0e3dc03c9d7/src/party.cpp#L438) (which is what really matter for us), or [this](https://github.com/raymondtfr/forgottenserver/blob/bd6c492d60f00bbb258042f5a6ffd0e3dc03c9d7/src/party.cpp#L443), or [this](https://github.com/raymondtfr/forgottenserver/blob/bd6c492d60f00bbb258042f5a6ffd0e3dc03c9d7/src/party.cpp#L450), or even [this](https://github.com/raymondtfr/forgottenserver/blob/bd6c492d60f00bbb258042f5a6ffd0e3dc03c9d7/src/party.cpp#L455).

So we could be sending that message description even if the member had the required level to share experience within a party.